src/common/meta/src/key.rs

```diff
@@ -77,7 +77,7 @@ lazy_static! {
 
 lazy_static! {
     static ref TABLE_NAME_KEY_PATTERN: Regex = Regex::new(&format!(
-        "^{TABLE_NAME_KEY_PREFIX}/({ALPHANUMERICS_NAME_PATTERN})/({ALPHANUMERICS_NAME_PATTERN})/({ALPHANUMERICS_NAME_PATTERN})$"
+        "^{TABLE_NAME_KEY_PREFIX}/({TABLE_NAME_PATTERN})/({TABLE_NAME_PATTERN})/({TABLE_NAME_PATTERN})$"
```

Comment:
Do catalog/schema/table need their own patterns?

Reply:
I think we can keep it same with the table name pattern. It's naturally to think the naming restrictions - about catalog, schema and table - are equal.